plugins/autosuspend.koplugin/main.lua

```diff
@@ -200,7 +208,15 @@ function AutoSuspend:addToMainMenu(menu_items)
     if not (Device:canPowerOff() or Device:isEmulator()) then return end
     menu_items.autoshutdown = {
         sorting_hint = "device",
-        text = _("Autoshutdown timeout"),
+        text_func = function()
+            if self.autoshutdown_timeout_seconds  then
+                -- @translators `h` stands for hours
+                return T(_("Autoshutdown timeout: %1 h"), self.autoshutdown_timeout_seconds / 3600)
```

Comment:
Here and around: will we see `Autoshutdown timeout: 0.333333333 ` if I set it to 20mn (can I?) ? We have some function in util.lua that displays a time/duration in a pretty 0h20m way.

Reply:
No, you can't enter a fraction of an hour in the spinwidget. But using our nice time-function better (then we get rid of the "h" and the "m" in the suspend menu).
Would it be a good idea to change the spinwidget to a datetimewidget here? So the user could enter "1:30" instead of "90"?

Reply:
Yes, we need definitively a datetimewidget here, as the autoshutdown uses a fractional hour yet: "1.25" means "1:15".

Reply:
I will do that on a different PR, as this one is just dealing with the menu.